.env

```
@@ -1,20 +1,21 @@
 # Clerk
-NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_d2VhbHRoeS1ibHVlZ2lsbC04MC5jbGVyay5hY2NvdW50cy5kZXYk
-CLERK_SECRET_KEY=sk_test_CtWAAZ4vgFFigtSLDCMy7IZx3C26Djbj9ihMXveUKo
-NEXT_PUBLIC_CLERK_SIGN_IN_URL=/sign-in
-NEXT_PUBLIC_CLERK_SIGN_UP_URL=/sign-up
-NEXT_CLERK_SIGN_IN_FALLBACK_URL=/
-NEXT_CLERK_SIGN_UP_FALLBACK_URL=/
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY='pk_test_d2VhbHRoeS1ibHVlZ2lsbC04MC5jbGVyay5hY2NvdW50cy5kZXYk'
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)